src/main/kotlin/calculator/view/OutputView.kt

```diff
@@ -0,0 +1,11 @@
+package calculator.view
+
+class OutputView {
+    fun showResult(result: Int) {
+        println("결과 : $result")
+    }
```

Comment:
tmi긴 한데 이렇게 단일표현식도 가능합니다!
```suggestion
    fun showResult(result: Int) = println("결과 : $result")
```